ferrocene/doc/requirements/src/conf.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+project = "Ferrocene Requirements"
+copyright = "The Ferrocene Developers"
+author = "The Ferrocene Developers"
+
+extensions = [
+    "ferrocene_autoglossary",
+    "ferrocene_qualification",
+    "ferrocene_toctrees",
+]
+
+ferrocene_id = "REQ"
```

Comment:
If the document is called `Ferrocene Requirements` the ID should be `FR`.

Reply:
d7864adc575